tests/test_functional_test.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from defense_finder_cli.main import run
+from tests import TooledTest, FakeExitCodeException
+
+
+def cleaned_lines(expected):
+    return set(sorted([line.upper() for line in expected.readlines()]))
```

Comment:
Why upper() ? I would remove it (tests don't currently pass because of this I think)

Reply:
When developing the tests, I saw some line where only the case differ, I though the case should not make the test fails but it is clearly open for debate ! `cleaned_lines` is used for both expected results and produced results, so using upper does not fails tests, it silences case issue.

Do you want the test to fail if the strain is written in different case ?

Reply:
Hi @jeanrjc 
Do you want the test to be case sensitive or not ? For now they are not